### PR TITLE
[WIPTEST] Add new test to remove dialog element

### PR DESCRIPTION
**Pull request by niyazRedhat**
{{pytest: cfme/tests/automate/test_service_dialog.py::test_remove_dialog_element }}

Following are the steps 
1. Add service dialog with fields
2. Edit service dialog editor
3. Remove any dialog field/element and save dialog

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit b3b8bcdd5e9f0b0a5a2ae4dbe40c6531bb822fe5:

`widgetastic_manageiq/__init__.py`:
- :bangbang: Line 1405:17: **E126** *continuation line over-indented for hanging indent*
- :bangbang: Line 1407:13: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by niyazRedhat** [comment]
Locally passing

File: cfme/tests/automate/test_service_dialog.py [hunk 1]
```diff
@@ -239,3 +239,33 @@ def test_radiobutton_dialog_element(appliance, request):
     }
     dialog, element = create_dialog(appliance, element_data)
     request.addfinalizer(dialog.delete_if_exists)
+
+
+def test_remove_dialog_element(appliance, request):
```
Comment (sshveta) [inline cfme/tests/automate/test_service_dialog.py]: please add a doc string to explain the test case and mention the BZ number that you are automating.

**Comment by niyazRedhat** [comment]
closing this PR. later I will raise new with changes

**Pull request closed without merging**